distros/fedora/fedora-install.sh

```
@@ -0,0 +1,659 @@
+        wget \
+        stow \
+        make \
+        gcc \
+        gcc-c++ \
+        cmake \
+        pkgconf-devel
+    
+    # Enable RPM Fusion repositories
+    log_info "Enabling RPM Fusion repositories..."
+    sudo dnf install -y \
+        https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)